bundle/manifests/patterns-operator.clusterserviceversion.yaml

```diff
@@ -363,4 +362,4 @@ spec:
   provider:
     name: Red Hat
     url: https://www.redhat.com/en/products/edge/validated-patterns
-  version: 0.0.17
+  version: 6.6.45
```

Comment:
Deliberate?

Reply:
Yes.  We change the version deliberately for testing.  This should not be committed.

Reply:
Fixed in latest commit.